src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,57 @@
+package model;
+
+public class FruitTransaction {
```

Comment:
According to the checklist, you should not keep all logic in a single package. Consider placing this model class in a 'model' package to better organize your project structure.